packages/flutter_tools/lib/src/base/process.dart

```diff
@@ -164,6 +164,16 @@ String runCheckedSync(List<String> cmd, {
   );
 }
 
+/// Run cmd and return stdout on success.
+///
+/// Throws the standard error output if cmd exits with a non-zero value.
+String runSyncAndThrowStdErrOnError(List<String> cmd) {
```

Comment:
we really should avoid adding more ways to run subprocesses... I don't suppose I can get you to embrace a yak shave that involves cleaning up this file, can I? :-)